backend/ctgov/script/etl.py

```diff
@@ -0,0 +1,161 @@
+import psycopg2
+import re
+
+from datetime import datetime
+from dotenv import load_dotenv
+from os import getenv
+from os.path import abspath
+from psycopg2 import Error
+
+dotenv_path = abspath("../../core/.env")
+print(dotenv_path)
+load_dotenv(dotenv_path)
+
+
+def print_current_time(type):
```

Comment:
since type is a keyword in python, if you could choose another variable name here, that would be better